pyogrio/_ogr.pyx

```diff
@@ -1,9 +1,13 @@
+<<<<<<< HEAD
```

Comment:
The conflicts were not resolved properly.

Reply:
whoops